cmd/instance/instance_create_test.go

```diff
@@ -0,0 +1,113 @@
+package instance_test
```

Comment:
there is a lot of replicated code here, everything in a for loop would be better